pyproject.toml

```
@@ -1,14 +1,77 @@
+[tool.flake8]
+max-line-length = 130
+ignore = [
+    "E203",  # whitespace before ':' - doesn't work well with black
+    "E402",  # module level import not at top of file
+    "E501",  # line too long - let black worry about that
+    "W503",  # line break before binary operator
+    "E226",  # missing whitespace around arithmetic operator
+]
 [build-system]
-requires = ["setuptools>=39.2.0", "wheel"]
+requires = ["setuptools>=61.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "monetio"
+version = "0.1.0"  # Update as needed or use dynamic versioning
```

```suggestion
dynamic = ["version"]
```

Also to specify the source: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata

---

I believe still need to add later in the file
```toml
[tool.setuptools.dynamic]
version = {attr = "monetio.__version__"}
```
so it knows where to get the version from